manifests/g/Google/GoogleDrive/79.0.2.0/Google.GoogleDrive.locale.en-US.yaml

```
@@ -0,0 +1,38 @@
+# Created with YamlCreate.ps1 v2.2.9 $debug=QUSU.CRLF.7-3-6.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Google.GoogleDrive
```

```suggestion
PackageIdentifier: Google.Drive
```
Please update file path to match this change.